OWML.ModHelper/Logging/ConsoleUtils.cs

```diff
@@ -0,0 +1,65 @@
+﻿using OWML.Common;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+
+namespace OWML.ModHelper
+{
+    public static class ConsoleUtils
+    {
+        public static readonly List<String> ErrorList = new List<String> {
+            "error",
+            "exception"
+        };
+
+        public static readonly List<String> WarningList = new List<String> {
+            "warning",
+            "disabled"
+        };
+
+        public static readonly List<String> SuccessList = new List<String> {
+            "success"
+        };
+
+        public static void WriteByType(MessageType type, string line)
+        {
+            switch (type)
+            {
+                case MessageType.Error:
+                    Console.ForegroundColor = ConsoleColor.Red;
+                    break;
+                case MessageType.Warning:
+                    Console.ForegroundColor = ConsoleColor.Yellow;
+                    break;
+                case MessageType.Success:
+                    Console.ForegroundColor = ConsoleColor.Green;
+                    break;
+                case MessageType.Message:
+                    Console.ForegroundColor = ConsoleColor.Gray;
+                    break;
+                case MessageType.Info:
+                    Console.ForegroundColor = ConsoleColor.Cyan;
+                    break;
+            }
+            Console.WriteLine(line);
+            Console.ForegroundColor = ConsoleColor.Gray;
+        }
+
+        public static MessageType ContentsToType(string line)
+        {
+            if (ErrorList.Contains(line.ToLower()))
```

Comment:
if you do this:

```
private bool Contains(string line, params string[] keyWords)
{
return keywords.Any(keyWord => line.Contains);
}
```
and use it like this:
`if (Contains(line, "error", "warning"))`
then you can get rid of the lists